### PR TITLE
Lunr local searcher implemented

### PR DESCRIPTION
**Pull request by iArchitSharma**
With this PR Layer5 Docs shift to the built-in local search by Docsy i.e. **Lunr**

[recording.webm](https://github.com/layer5io/docs/assets/74408634/bc36c7a3-dd63-4029-989e-f6d34606ad5a)

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by iArchitSharma** [comment]
Thanks @leecalcote

**Comment by ritiksaxena124** [comment]
@iArchitSharma looks good. You see on hover over the title of the search results the text is not visible.

**Comment by leecalcote** [comment]
Recently, I have added a new shortcut key to the search field.

**Review by leecalcote — CHANGES_REQUESTED** [review]
This PR overwrites the recent keyboard shortcut for search that I added.

**Comment by iArchitSharma** [comment]
@leecalcote fixed

**Comment by leecalcote** [comment]
@iArchitSharma thank you. Will you implement a clickaway for closing the search panel (clickaway and ESC might be nice, too)

File: assets/scss/_search.scss [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// Search
```
Comment (leecalcote) [inline assets/scss/_search.scss:1]: This CSS duplicates classes in the assets/scss/_styles_project.scss file. Will you de-duplicate and consolidate all search-related CSS into this file, @iArchitSharma?

Comment (iArchitSharma) [reply assets/scss/_search.scss:1]: actually those search classes are used for styling different search bar as the search bar on top right side of nav is different than the search bar on the sidebar nav of docs.
by consolidating both of them on same will make them look same.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: assets/scss/_search.scss (hunk 1 above)
Comment (iArchitSharma) [reply assets/scss/_search.scss:1]: Done made the changes

**Review by leecalcote — CHANGES_REQUESTED** [review]
/ hotkey does not work/

**Pull request merged**